app/(tabs)/summary/main-summary.tsx

```
@@ -0,0 +1,153 @@
+                }
+                rippleColor={c.color}
+                mode="outlined"
+                showSelectedCheck={false}
+                icon={undefined}
+                style={{margin: 4}}
+                selected={!!filters.find((f) => f.id === c.id)}
+                onPress={handleFilters(c.id)}>
+                <Text
+                  style={{
+                    fontSize: 14,
+                    fontWeight: !!filters.find((f) => f.id === c.id)
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)